release-notes/kvm/v9.0.1.md

```diff
@@ -0,0 +1,103 @@
+## :zap: Giant Swarm Release 9.0.1 for KVM :zap:
+
+This release includes Kubernetes v1.15.11 as well as some reliability and user experience improvements.
+We highly recommend you to upgrade to this release if you want to continue running on Kubernetes 1.15 for now.
+This is also the first release which is internally represented by our [Release CRD](https://docs.giantswarm.io/reference/cp-k8s-api/releases.release.giantswarm.io/). This is done in preparation of opening up the control plane to you.
+
+In addition, this release replaces CoreOS with Flatcar Container Linux.
+CoreOS has gone [end-of-life](https://coreos.com/os/eol/) and is being rapidly phased out.
+Flatcar is a compatible fork of CoreOS which receives ongoing support.
+To continue receiving security updates and to minimize the effort needed to migrate in the future, we recommend upgrading to this release.
+
+This release includes multiple improvements to the NGINX Ingress Controller app:
+1. It upgrades to upstream ingress-nginx `v0.30.0`.
+2. Optional metrics service for prometheus-operator support was added. This allows prometheus-operator to scrape metrics to monitor the app.
+3. NGINX IC SSL support for old browsers and clients (e.g. Safari 9) is dropped. This is due to default SSL ciphers no longer including AES-CBC based ciphers since they are considered weak. At your own risk, weak ciphers can still be enabled on demand independently for each cluster.
+
+Below, you can find more details on components that were changed with this release.
+
+### Kubernetes [v1.15.11](https://github.com/kubernetes/kubernetes/blob/master/CHANGELOG/CHANGELOG-1.15.md#changelog-since-v11510)
```

Comment:
Would it also make sense to include a changelog for `containerlinux`?
https://github.com/giantswarm/releases/blob/master/release-notes/kvm/v9.0.0.md#coreos-container-linux-v219150

Reply:
Yes good idea - included